src/python/pants/backend/project_info/dependencies_test.py

```diff
@@ -209,3 +215,165 @@ def test_python_dependencies(rule_runner: PythonRuleRunner) -> None:
         ],
         closed=True,
     )
+
+
+def test_python_dependencies_output_format_json_direct_deps(rule_runner: PythonRuleRunner) -> None:
+    create_python_requirement_tgts(rule_runner, "req1", "req2")
+    create_python_sources(rule_runner, "dep/target")
+    create_python_sources(
+        rule_runner, "some/target", dependencies=["dep/target", "3rdparty/python:req1"]
+    )
+    create_python_sources(
+        rule_runner,
+        "some/other/target",
+        dependencies=["some/target", "3rdparty/python:req2"],
+    )
+
+    assert_deps = partial(assert_dependencies, rule_runner)
```

Comment:
```suggestion
    assert_deps = partial(assert_dependencies, rule_runner, output_format=DependenciesOutputFormat.json)
```

can then remove that arg from all calls below.

Reply:
Excellent idea, this is now done.